lib/core/utils/get-xpath.js

```diff
@@ -50,9 +50,10 @@ function getXPathArray(node, path) {
 		var element = {};
 		element.str = node.nodeName.toLowerCase();
 		// add the id and the count so we can construct robust versions of the xpath
-		if(node.getAttribute && node.getAttribute('id') &&
-			node.ownerDocument.querySelectorAll('#' + axe.utils.escapeSelector(node.id)).length === 1) {
-
+		if(
```

Comment:
Can we put the first part of the expression inline with the `if(` for consistency's sake?